src/Compilers/Core/Portable/Operations/OperationInterfaces.xml

```diff
@@ -831,7 +831,7 @@
       </Comments>
     </Property>
   </Node>
-  <Node Name="IUnaryOperation" Base="IOperation" SkipInVisitor="true">
+  <Node Name="IUnaryOperation" Base="IOperation" VisitorName="VisitUnaryOperator">
```

Comment:
why does the visitor name not match the operation?

Reply:
Because it wasn't generated originally, and slipped through the cracks. So, we're stuck with the public api we have.